README.md

```diff
@@ -10,19 +10,38 @@ Inspired by a pattern seen in [Polar](http://www.polarb.com/), [IE 10+](http://i
 
 The plugin works in any browser that supports resetting the `type` attribute of `<input>` elements (pretty much everything newer than IE8). The plugin should fall back gracefully in cases where this is not supported.
 
-If [Bower](http://bower.io/)'s your thing, you can install this plugin by running `bower install hideShowPassword` in your project directory.
+## Installation
 
-## Dependencies
+Simply include the plugin after you've included [jQuery](http://jquery.com/):
```

Comment:
Hahaha. Personal trifle here, but I dislike when the word "Simply" is involved with instructions because if something goes wrong, I feel like a *moron*.